packages/unpublished-fastboot-test-app/package.json

```diff
@@ -16,7 +16,7 @@
     "lint:js": "eslint --config ../../.eslintrc.js --ignore-path ../../.eslintignore .",
     "start": "ember serve",
     "test": "ember test --test-port=0",
-    "test:try-one": "ember try:one"
+    "test:scenario": "ember try:one"
```

Comment:
I renamed this just because we have the other command running `try-one` in all of the main packages but this one has it's own config.

Reply:
Do we not use this in CI?

Reply:
We do but we use it via `yarn test:fastboot` in the root package.json and I also renamed it there.